packages/compass-e2e-tests/helpers/commands/instance-workspaces.ts

```diff
@@ -5,9 +5,8 @@ export async function navigateToInstanceTab(
   browser: CompassBrowser,
   tabName: 'My Queries' | 'Performance' | 'Databases' = 'My Queries'
 ): Promise<void> {
-  const sidebarNavigationItem = browser.$(
-    Selectors.sidebarInstanceNavigationItem(tabName)
-  );
+  const sidebarNavigationItem =
```

Comment:
Again - sidebarNavigationItem was a promise that would resolve to an element which is probably not what we want to pass to clickVisible.

Reply:
This is specifically not awaited so that clickVisible can re-request element from a selector instead of trying to click an element that was potentially removed / reinserted in DOM. Like an element matching `.foo` is in the DOM, but awaiting here would keep a reference to a specific DOM node that is gone already. In webdriver types this is referred to as `ChainablePromiseElement`. Awaiting a specific element and passing it around was causing issues in a few places, so this is why clickVisible was extended to support this. If it's not an issue anymore, feel free to remove it, but then you'd probably want to remove the ability to pass `ChainablePromiseElement` to the method itself too

Reply:
I relented and just added ChainalePromiseElement support everywhere.

Reply:
Btw this is usually why I just pass a selector and would re-select based on the selector. Otherwise you inevitably end up with a stale element eventually. I'm not 100% sure if ChainablePromiseElement won't end up caching the element id internally somehow anyway.

Also we have sooo many webdriver warnings flying past about stale elements in version 8 now.. 😬